cmd/update-notifier.go

```diff
@@ -28,25 +28,35 @@ import (
 	"github.com/fatih/color"
 )
 
+// computeUpdateMessage - calculates the update message, only if a
+// newer version is available.
+func computeUpdateMessage(downloadURL string, oldVersion, newVersion time.Time) (string, bool) {
+	if !newVersion.After(oldVersion) || downloadURL == "" {
+		return "", false
+	}
+	// Last argument below is never actually used as (newVersion >
+	// oldVersion).
+	newerThanStr := humanize.RelTime(oldVersion, newVersion, "ago", "huh?")
```

Comment:
This `huh?` is not nice.. We need to find something else. In-fact when this happens we can throw an error i.e `return "", false` which kind of makes sense to me that no update is needed.. It can happen so that users can build it themselves using `make release` .